<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimalProfile</class>
 <widget class="QDialog" name="AnimalProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AnimalProfile{
	border-image: url(:/img/istockphoto-1163775073-612x612.jpg);
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>701</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
background-color: rgb(255, 222, 139);
border-radius:30px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#groupBox_2{border-image: url(:/img/cat-logo.webp);border-radius:80px;}
</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="IdLabel">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="SpecieLabel">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Specie:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="SpecieRaspuns">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="SexLabel">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Sex:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="SexRaspuns">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="VarstaLabel">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Varsta:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="VarstaRaspuns">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="LocatieLabel">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Locatie:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="LocatieRaspuns">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="TalieLabel">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Talie:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="TalieRaspuns">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="IDRaspuns">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton{
border-radius:10px;
	background-color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>Adopta</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI Black</family>
           <pointsize>11</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#pushButton_2{
border-radius:10px;
	background-color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
